packages/cli/.prettierrc.yml

```diff
@@ -0,0 +1,5 @@
+tabWidth: 2
```

Comment:
can we use the default prettier config at project root instead

Reply:
Will do it 👍